src/core/docs/api_details.md

```diff
@@ -0,0 +1,30 @@
+# OpenVINO Core API
+
+OpenVINO Core API contains two folders:
+ * [ngraph](../include/ngraph/) - is a legacy API, this API is not developed more. Only aliases to new operation and operation sets extend this API.
```

Comment:
is not developed more ---> is no longer being developed
maybe "aliases to new operationS"?